gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -317,8 +317,11 @@ abstract class KspAATask @Inject constructor(
                         val konanTargetName = kotlinCompilation.target.konanTarget.name
                         cfg.konanTargetName.value(konanTargetName)
                         cfg.konanHome.value((kotlinCompileProvider.get() as KotlinNativeCompile).konanHome)
+
+                        val isKlibCrossCompilationEnabled =
+                            project.findProperty("kotlin.native.enableKlibsCrossCompilation") == "true"
```

Comment:
This is not the best approach. In the presence of custom CInterops cross compilation still be disabled even if this Gradle property value is `true`.

@ting-yuan could you file a new Kotlin issue to expose some API to get the cross-compilation state?

Reply:
Created here: https://youtrack.jetbrains.com/issue/KT-77258/Query-status-of-cross-compilation-or-compilation-task

Thank you!